movies/2018/ocean-8.json

```diff
@@ -0,0 +1,25 @@

```

Comment:
This summary is a little too long.   Could you please  use the following:

"Debbie Ocean gathers an all-female crew to attempt an impossible heist at New York City's yearly Met Gala."